### ISSUE TITLE
`noteBackground` in `sequenceDiagram` is not adjustable

### ISSUE THREAD

**Issue by JKRhb** [hikerpig/pintora#272]
Hi there :)

When using the `themeConfig` in a `sequenceDiagram`, it seems as if the `noteBackground` color is not adjustable. Instead, it is always rendered yellow. You can find a minimal example for reproducing the issue below.

# Source code

```
sequenceDiagram
  @config({
     "themeConfig": {
       "themeVariables": {
         "noteBackground": "#ff0000"
       }
     }
  })
  A->>B: Hi, this is a minimal example.

  @note over A,B: My background is not red :(
```

# Rendered sequence diagram

<img width="442" alt="image" src="https://github.com/hikerpig/pintora/assets/12641361/0f327dc6-b266-4266-a49e-6ad4c2a01956">

**Comment by hikerpig** [comment]
Ah this is because `@config` can only set diagram local config but not global theme config for now. You will have to call `pintora.setConfig` JS API to [alter global theme](https://pintorajs.vercel.app/docs/configuration/theme/).
But I will look into it, it's reasonable that one should be able to alter diagram local theme through `@config` directive.

**Comment by hikerpig** [comment]
Fixed in v0.7.4
